src/EditorFeatures/CSharpTest/UseExpressionBody/Refactoring/UseExpressionBodyForLocalFunctionsRefactoringTests.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp.UseExpressionBody;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.UseExpressionBody
+{
+    [WorkItem(31477, "https://github.com/dotnet/roslyn/issues/31477")]
```

Comment:
enh.....................

Reply:
Maybe I'm missing something obvious (especially since I'm pretty tired today), but what exactly do you mean here?

Reply:
Maybe I'm missing something obvious (especially since I'm pretty tired today), but what exactly do you mean here?

Reply:
You apply WorkItem on test methods, not classes. Hard guess.

Reply:
I don't particularly like it because what you're doing now is marking this attribute on all *future* additions to this class.  It makes the attribute overly broad, instead of targetting specific tests.

Furthermore, we don't generally use workitem for *new* feature work.  It's generally for real 'bugs'.

Reply:
regardless, roslyn doesn't follow this pattern elsewhere, so i would prefer we not do things differently just because it's 'legal'.  Please follow the "when in rome" approach for IDE changes.  thanks!

Reply:
Well, #31477 is labelled as a _bug_, even though I would personally also call it a feature request. I see your point, I'll just remove the attributes from both classes.